core/core-security/src/main/java/com/pgms/coresecurity/security/jwt/booking/BookingExceptionHandlerFilter.java

```diff
@@ -40,11 +43,11 @@ protected void doFilterInternal(HttpServletRequest request, HttpServletResponse
 		}
 	}
 
-	private void sendFailResponse(HttpServletResponse response, BookingErrorCode bookingErrorCode) throws IOException {
+	private void sendFailResponse(HttpServletResponse response, BaseErrorCode bookingErrorCode) throws IOException {
```

Comment:
여기도 HttpResponseUtil 적용할 수 있을 거 같습니다! 

```suggestion
	private void sendFailResponse(HttpServletResponse response, BaseErrorCode bookingErrorCode) throws IOException {@Component
@Slf4j
public class BookingExceptionHandlerFilter extends OncePerRequestFilter {
	@Override
	protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response,
		FilterChain filterChain) throws IOException {
		try {
			filterChain.doFilter(request, response);
		} catch (SecurityCustomException e) {
			BaseErrorCode bookingErrorCode = e.getErrorCode();
			HttpResponseUtil.setErrorResponse(response, bookingErrorCode.getStatus(), bookingErrorCode.getErrorResponse());
		} catch (Exception e) {
			log.error(e.getMessage(), e);
			BookingErrorCode bookingErrorCode = BookingErrorCode.INTERNAL_SERVER_ERROR;
			HttpResponseUtil.setErrorResponse(response, bookingErrorCode.getStatus(), bookingErrorCode.getErrorResponse());
		}
	}
}

```

Reply:
수정했습니다!